com.unity.ml-agents/CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ and this project adheres to
 
 ## [Unreleased]
 ### Major Changes
+### Minor Changes
+### Bug Fixes
+
+
+## [2.0.0-preview] - 2021-04-22
```

Comment:
This might need to be `2.0.0-exp.1` (since that's what's here):
https://github.com/Unity-Technologies/ml-agents/blob/f94a75ab1bb0f2a79fc55740fefa136fb1993933/com.unity.ml-agents/package.json#L4